src/sphinx/reference/event-sourcing.rst

```diff
@@ -275,6 +280,9 @@ The ``destination`` confirms the delivery of the message by sending a ``Confirma
 
 When the actor is re-started, unconfirmed reliable messages are automatically re-delivered to their ``destination``\ s. The example actor additionally schedules ``redeliverUnconfirmed`` calls to periodically re-deliver unconfirmed messages. This is done within the actor’s command handler.
 
+.. note::
+   In the above example a pattern guard is used to guarantee that each confirmation is processed once-only by ensuring that the ``deliveryId`` of the ``Confirmation`` message is still unconfirmed. This pattern may only be applied if the ``stateSync`` member of the ``EventsourcedActor`` is set to ``true``. For further details on ``stateSync`` see section :ref:`state-sync`.
```

Comment:
> In the above example a pattern guard is used to guarantee that each confirmation is processed once-only ...

Rephrase to: *In the above example a pattern guard is used for idempotent confirmation processing ...* ?